README.md

```diff
@@ -42,15 +42,15 @@ Optimas is a Python library designed for highly scalable optimization, from lapt
 ## Installation
 You can install Optimas from PyPI (recommended):
 ```sh
-pip install optimas
+python3 -m pip install optimas[all]
 ```
 from conda-forge:
 ```sh
 conda install optimas --channel conda-forge
 ```
 or directly from GitHub:
 ```sh
-pip install git+https://github.com/optimas-org/optimas.git
+python3 -m pip install "optimas[all] @ git+https://github.com/optimas-org/optimas.git"
```

Comment:
Do you think we could use `python` instead of `python3` so that it also works on windows?

Reply:
Oh, do the Windows installers not add a `python3` alias?
https://docs.python.org/3/using/windows.html

With installer do you see this issue on?